core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go

```diff
@@ -446,9 +463,32 @@ lookbackBlocks         	= %d # This is an integer
 		t.Logf("TestRedeemRandomness 2nd word err: %+v", err2)
 		rw3, err3 := uni.consumer.SReceivedRandomnessByRequestID(nil, big.NewInt(1), big.NewInt(0))
 		t.Logf("FulfillRandomness 1st word err: %+v", err3)
-		t.Log("randomness from redeemRandomness:", rw1.String(), rw2.String())
-		t.Log("randomness from fulfillRandomness:", rw3.String())
-		return err1 == nil && err2 == nil && err3 == nil
+		rw4, err4 := uni.batchConsumer.SReceivedRandomnessByRequestID(nil, big.NewInt(2), big.NewInt(0))
```

Comment:
Out of curiosity, do we need this many `err*` variables? One idea using `multierr.Append`:

```golang
var errs error
rw1, err := uni.consumer.SReceivedRandomnessByRequestID(nil, big.NewInt(0), big.NewInt(0))
errs = multierr.Append(errs, err)
rw2, err := uni.consumer.SReceivedRandomnessByRequestID(nil, big.NewInt(0), big.NewInt(1))
errs = multierr.Append(errs, err)
// ...
return errs == nil
```

Reply:
yeah that's a nice extra step to clean this up.